src/dictionaries/words

```diff
@@ -209,6 +209,7 @@ toolbar
 tuple
 uiserver
 unparsed
+unpause
 unpaused
```

Comment:
```suggestion
```
IMO we should be consistently using the word "resume", not "unpause"

Reply:
Objection - Hilary has used this word in a way which clarifies rather than obfuscates.

Reply:
IMO both are fine.